### PR TITLE
fix: applying default brandColor in embeds

### LINKED ISSUE #10497: [CAL-4740] Embed brand colors doesn't work properly
Found a bug? Please fill out the sections below. 👍

### Issue Summary

The brand color set in the Settings>Appearance is not set automatically in the brand color in embeds. When I try to set brand color from the color picker, it is set only for one theme, either dark or light. 

### Steps to Reproduce

1. Set your brand colors in Settings>Appearance.
2. Open embeds from an event. Select inline embeds.
3. The brand color is not selected according to the colors set in Settings.
4. When I select the brand color, it is only visible in preview for one theme, either light or dark.

Any other relevant information. For example, why do you consider this a bug and what did you expect to happen instead?

### Actual Results

Brand color is not being set properly

### Expected Results

I shouldn't have to set brand colors in the embeds as it's already set in the settings. If the option to set brand colors is still required, the selected value should be according to the colors selected in the settings and should be available for both the themes.

### Technical details

https://app.birdeatsbug.com/sessions/N2G-znF0x0ipbn--ofl88su--GlD613Iqo31PVJsPl7G

### Evidence

Manual testing. Video recording attached above

<sub>[CAL-4740](https://linear.app/calcom/issue/CAL-4740/embed-brand-colors-doesnt-work-properly)</sub>

### PR DESCRIPTION
**Pull request by pratyush3124**
## What does this PR do?

Fixes #10497
Now the brand color that the user sets in Settings>Appearance will be default brand color in embeds.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## How should this be tested?
1. Set a brand color in Settings>Appearance.
2. Go to Event Types, and click on embed of any event.
3. The brand color here should be the one user selects in settings.

Images-
s1 s2

## Mandatory Tasks

- [x] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

### CODE REVIEW

**Comment by pratyush3124** [comment]
Hey, this is my first opensource contribution, pls review this PR.

File: packages/features/embed/Embed.tsx [hunk 1]
```diff
@@ -545,6 +545,9 @@ const EmbedTypeCodeAndPreviewDialogContent = ({
     { enabled: !!eventId && embedType === "email", refetchOnWindowFocus: false }
   );
 
+  const { data: user } = trpc.viewer.me.useQuery();
```
Comment (joeauyeung) [inline packages/features/embed/Embed.tsx:548]: @hariombalhara do we want to send the whole `/me` endpoint to the embed component?

**Comment by joeauyeung** [comment]
> Hey, this is my first opensource contribution, pls review this PR.

Thanks for your contribution @pratyush3124. 

Just a heads up you can assign reviewers to your PR. This way they will get a notification to review your PR.

CleanShot 2023-09-08 at 15 31 17@2x

File: packages/features/embed/Embed.tsx (hunk 1 above)
Comment (hariombalhara) [reply packages/features/embed/Embed.tsx:536]: Thanks for highlighting this @joeauyeung. First of all, we shouldn't do it here. There are wrapper components that provide all the data to Embed, EventTypeEmbed.tsx and RoutingFormEmbed.tsx. They should be responsible to do that and pass a `defaultBrandColor` prop

I think it's okay to do reuse viewer.me query as it's anyways fetched at a lot of places.

**Review by hariombalhara — CHANGES_REQUESTED** [review]
Thank you @pratyush3124 🙏 It just needs a small change.

**Pull request closed without merging**